EX2/6.2_2000_run.py

```diff
@@ -0,0 +1,19 @@
+import time
+
+
+def timer(f, *arguments):
+    start_time = time.time()
+    map(f, arguments)
```

Comment:
This is not the relevant task - you should check the function executions time for a single execution with all the arguments, not an execution for every argument
![image](https://github.com/Lev-Excellenteam-2023/git-pull-request-AsherMentzer/assets/142482033/d141bcd6-f84a-4a5a-9b36-8f745147b8fb)